### ISSUE TITLE
Error template does not render correctly on Windows

### ISSUE THREAD

**Issue by headius** [bundler/bundler#3572]
I had the misfortune of hitting some error during a bundle install, and bundler's error template displayed but did not render well.

Here is slightly different output from isolating the heredoc bundler uses into a separate file: https://gist.github.com/headius/6caff73de251ef425740

The problem with the rendering stems from the use of unicode "horizontal bar" characters for the header and footer. On Windows, these are not rendering correctly, probably due to differences in the platform's default encoding and MRI's default UTF-8 encoding.

I'm not sure if there's an MRI issue here preventing the template from rendering in the proper encoding, but at the very least I think bundler should use only 7-bit characters in its error message and template.

**Comment by headius** [comment]
Here's the full content:

```
λ bundle install                                                                                                                                                                                                
ΓÇòΓÇòΓÇò ERROR REPORT TEMPLATE ΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇò           
- What did you do?                                                                                                                                                                                              
- What did you expect to happen?                                                                                                                                                                                
- What happened instead?                                                                                                                                                                                        
                                                                                                                                                                                                                
Error details                                                                                                                                                                                                   
                                                                                                                                                                                                                
    Errno::ENOENT: No such file or directory - nul                                                                                                                                                              
    org/jruby/RubyFile.java:342:in `initialize'                                                                                                                                                                 
    org/jruby/RubyIO.java:1124:in `open'                                                                                                                                                                        
    C:/Users/headius/jruby/lib/ruby/stdlib/rubygems/user_interaction.rb:690:in `initialize'                                                                                                                     
    C:/Users/headius/jruby/lib/ruby/gems/shared/gems/bundler-1.8.2/lib/bundler/ui/rg_proxy.rb:9:in `initialize'                                                                                                 
    C:/Users/headius/jruby/lib/ruby/gems/shared/gems/bundler-1.8.2/lib/bundler/cli.rb:24:in `initialize'                                                                                                        
    C:/Users/headius/jruby/lib/ruby/gems/shared/gems/bundler-1.8.2/lib/bundler/vendor/thor.rb:359:in `dispatch'                                                                                                 
    C:/Users/headius/jruby/lib/ruby/gems/shared/gems/bundler-1.8.2/lib/bundler/vendor/thor/base.rb:440:in `start'                                                                                               
    C:/Users/headius/jruby/lib/ruby/gems/shared/gems/bundler-1.8.2/lib/bundler/cli.rb:10:in `start'                                                                                                             
    C:/Users/headius/jruby/lib/ruby/gems/shared/gems/bundler-1.8.2/bin/bundle:20:in `block in (root)'                                                                                                           
    C:/Users/headius/jruby/lib/ruby/gems/shared/gems/bundler-1.8.2/lib/bundler/friendly_errors.rb:6:in `with_friendly_errors'                                                                                   
    C:/Users/headius/jruby/lib/ruby/gems/shared/gems/bundler-1.8.2/bin/bundle:18:in `<top>'                                                                                                                     
    org/jruby/RubyKernel.java:984:in `load'                                                                                                                                                                     
    C:\Users\headius\jruby\bin\bundle:23:in `<top>'                                                                                                                                                             
                                                                                                                                                                                                                
Environment                                                                                                                                                                                                     
                                                                                                                                                                                                                
    Bundler   1.8.2                                                                                                                                                                                             
    Rubygems  2.4.5                                                                                                                                                                                             
    Ruby      2.2.1p0 (2015-04-16 revision 49761) [java]                                                                                                                                                        
    Git       1.9.5.msysgit.0                                                                                                                                                                                   
ΓÇòΓÇòΓÇò TEMPLATE END ΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇòΓÇ
òΓÇòΓÇò                                                                                                                                                                                                         
                                                                                                                                                                                                                
Unfortunately, an unexpected error occurred, and Bundler cannot continue.                                                                                                                                       
                                                                                                                                                                                                                
First, try this link to see if there are any existing issue reports for this error:                                                                                                                             
https://github.com/bundler/bundler/search?q=No+such+file+or+directory+-+nul&type=Issues                                                                                                                         
                                                                                                                                                                                                                
If there aren't any reports for this error yet, please create copy and paste the report template above into a new issue. Don't forget to anonymize any private data! The new issue form is located at:          
https://github.com/bundler/bundler/issues/new                                                                                                                                                                   
```

**Comment by headius** [comment]
Note that the error is due to a bug in JRuby, but MRI renders this template very similarly.

**Comment by indirect** [comment]
Sounds like we should switch to = or - as separator characters. :(

**Comment by agis-** [comment]
Fixed by a5107e400555217d90b0abda9e72f9140aed4410.
